.github/workflows/on_pull_request.yml

```diff
@@ -8,8 +8,24 @@ on:
 
 # These jobs use fairly short names as they are a prefix in the display hierarchy
 jobs:
+  temp:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Show context
+        run: |
+          echo "GITHUB_CONTEXT": $GITHUB_CONTEXT
+          echo "JOB_CONTEXT": $JOB_CONTEXT
+          echo "INPUTS_CONTEXT": $INPUTS_CONTEXT
+          echo "ENV_CONTEXT": $ENV_CONTEXT
+        env:
+          ENV_CONTEXT: ${{ toJson(env) }}
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+          JOB_CONTEXT: ${{ toJson(job) }}
+          INPUTS_CONTEXT: ${{ toJson(inputs) }}
+
   checks:
     name: Checks
+    if: github.event.pull_request.head.repo.owner.login == 'rerun-io'
```

Comment:
These have to be on a per-job basis rather than just skipping the whole workflow?

Reply:
I think so, or at least I couldn't find a better way to do this. We need to know if the PR comes from within the `rerun-io` organization, and there doesn't seem to be a built-in way to do this on `pull_request` events.